src/sailfishusermanager.cpp

```diff
@@ -332,6 +334,24 @@ uint SailfishUserManager::addUser(const QString &name)
     return addSailfishUser(user, name);
 }
 
+void SailfishUserManager::executeScripts(uint uid, const QString &directory)
+{
+    QDir creation(directory, "*.sh", QDir::NoSort, QDir::Files | QDir::Executable);
```

Comment:
nit: Now that this is used for both create and remove "creation" is a bit of a misnomer.

Reply:
and how it's should be better written ?

Reply:
Well, I do not have an obvious candidate, just a bit irked to think removal operation would iterate creation list... It is a QDir by type but is used for locating script files -> for "create" purpose it could have been "creationScripts", and for removal "removalScripts" might have been appropriate -> now that it is generic enough to deal with both of those / something else too -> droping the parts that hint to specific purpose would leave: "scripts"?

Reply:
"Change execution of scripts to generic"

Reply:
I did not cache the previous PR version, so I can't figure out what changed.  Did you change the method name? I meant the variable "creation". In any case it was cosmetic to begin with, so feel free to leave it as it is now.